### PR TITLE
Maximum audio for custom acoustic models is now 200 hours.

### PR DESCRIPTION
**Pull request by jeffpk62**

### CODE REVIEW

File: acoustic-create.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ The method also accepts an optional `allow_overwrite` query parameter to overwri
...
 
-You can add any number of audio resources to a custom model by calling the method once for each audio or archive file. The addition of one audio resource must be fully complete before you can add another. You must add a minimum of 10 minutes and a maximum of 100 hours of audio that includes speech, not silence, to a custom model before you can train it. No audio- or archive-type resource can be larger than 100 MB. For more information, see [Guidelines for adding audio resources](/docs/services/speech-to-text/acoustic-resource.html#audioGuidelines).
+You can add any number of audio resources to a custom model by calling the method once for each audio or archive file. The addition of one audio resource must be fully complete before you can add another. You must add a minimum of 10 minutes and a maximum of 200 hours of audio that includes speech, not silence, to a custom model before you can train it. No audio- or archive-type resource can be larger than 100 MB. For more information, see [Guidelines for adding audio resources](/docs/services/speech-to-text/acoustic-resource.html#audioGuidelines).
```
Comment (scot-taylor-ibm) [inline acoustic-create.md]: Sigh. Does the link need to be in the new format (`/docs/services/speech-to-text?topic=speech-to-text-audioGuidelines` or somesuch)?

File: index.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ subcollection: speech-to-text
 # About
 {: #about}
 
-**Service update:** *The {{site.data.keyword.speechtotextshort}} service was updated on March 15, 2019. The service now supports audio in the A-law (`audio/alaw`) format. For more information, see the [15 March 2019 service update](/docs/services/speech-to-text/release-notes.html#March2019c) in the release notes*.
+**Service update:** *The {{site.data.keyword.speechtotextshort}} service was updated on April 3, 2019. The service now allows you to add a maximum of 200 hours of audio to a custom acoustic model. For more information, see the [3 April 2019 service update](/docs/services/speech-to-text/release-notes.html#April2019) in the release notes*.
```
Comment (scot-taylor-ibm) [inline index.md]: Same question about linking format.

File: acoustic-create.md (hunk 1 above)
Comment (jeffpk62) [reply acoustic-create.md]: Not until May.

File: index.md (hunk 2 above)
Comment (jeffpk62) [reply index.md]: See above.

**Pull request merged**